node_modules/oae-authentication/lib/rest.js

```diff
@@ -44,3 +43,93 @@ require('./strategies/twitter/rest');
  */
 OAE.globalAdminRouter.on('post', '/api/auth/logout', AuthenticationAPI.logout);
 OAE.tenantRouter.on('post', '/api/auth/logout', AuthenticationAPI.logout);
+
+/**
+ * Add two endpoints for REST control
+ */
+
+
+var _getResetPasswordSecret = function(req, res){
+
+	AuthenticationAPI.getResetPasswordSecret(req.ctx, req.params.tenantAlias, req.params.username, function(err, token){
+	    if (err) {
+	        return callback(err);
+	    }
+
+		return res.send(token);
+	})
+};
+
+/**
+ * @REST Globalresetinit
+ *
+ * generate an user token based on a existing username
+ *
+ * @Server      Admin
+ * @Method      GET
+ * @Path        /auth/local/reset/init/{tenatAlias}/{userId}
+ * @PathParam   {string}        userId     			The userId for the user
+ * @Return      {string}
+ * @HttpResponse                200                     Configuration value cleared
+ * @HttpResponse                400                     Missing configuration. Example configuration: `"oae-authentication/twitter/enabled"`
+ * @HttpResponse                400                     Missing tenant alias
+ * @HttpResponse                400                     You cannot mix clearing an entire element and an optionalKey
+ * @HttpResponse                401                     Only authorized tenant admins can change config values
+ * @HttpResponse                401                     User is not allowed to update config value ...
+ * @HttpResponse                404                     Config value ... does not exist
+ */
+OAE.globalAdminRouter.on('post', '/api/auth/local/reset/init/:tenantAlias/:userId', _getResetPasswordSecret);
```

Comment:
Are we sure we want to add this functionality for admin users as well as regular users?